Dockerfile

```diff
@@ -10,9 +11,8 @@ RUN go mod download
 
 COPY cmd cmd
 COPY internal internal
-COPY pkg pkg
 
-RUN go build -o /server ./cmd/main.go
+RUN go build -o /server ./cmd/main.go -tags=nomsgpac
```

Comment:
The tag should be nomsgpac**k**. Per `go help build` go has the following argument format, so the build flags should be just before package/file names.
```
usage: go build [-o output] [build flags] [packages]
```

Reply:
my bad, fixed it.